src/md/bylaws.md

```diff
@@ -0,0 +1,381 @@
+"http://www.apache.org/dev/committers.html">Committer FAQ</a> which provides more
+details on the requirements for Committers
+
+A committer who makes a sustained contribution to the project may be invited to
+become a member of the PMC. The form of contribution is not limited to code. It can
+also include code review, helping out users on the mailing lists, documentation,
+etc.
+
+
+###Project Management Committee
+
+The Project Management Committee (PMC) for Apache Apex was created by a
+resolution of the board of the Apache Software Foundation on ***TBD***. The
+PMC is responsible to the board and the ASF for the management and oversight of the
+Apache Apex codebase. The responsibilities of the PMC include:
+
+* Deciding what is distributed as products of the project. In particular all releases must be approved by the PMC
+* Maintaining the project&rsquo;s shared resources, including the codebase repository, mailing lists, websites.
+* Speaking on behalf of the project
+* Resolving license disputes regarding products of the project
+* Nominating new PMC members and committers
+* Maintaining these bylaws and other guidelines of the project
+
+Membership of the PMC is by invitation only and must be approved by a lazy
+consensus of active PMC members. A PMC member is considered
+&ldquo;emeritus/inactive&rdquo; by not contributing in any form to the project for
+over one year. An emeritus PMC member may request reinstatement to the PMC. Such
+reinstatement is subject to lazy consensus of active PMC members. A PMC member may
+resign their membership from the PMC by their own declaration. Membership of the
+PMC can be revoked by an unanimous vote of all the active PMC members other than
+the member in question.
+
+The chair of the PMC is appointed by the ASF board. The chair is an office
+holder of the Apache Software Foundation (Vice President, Apache Apex) and has
+primary responsibility to the board for the management of the projects within the
+scope of the Apex PMC. The chair reports to the board quarterly on developments
+within the Apex project. The PMC may consider the position of PMC chair annually,
+and if supported by a successful vote to change the PMC chair, may recommend a new
+chair to the board. Ultimately, however, it is the board&rsquo;s responsibility who
+it chooses to appoint as the PMC chair.
+
+##Decision Making
+
+Within the Apex project, different types of decisions require different forms of
+approval. For example, the previous section describes several decisions which require
+&ldquo;lazy consensus&rdquo; approval. This section defines how voting is performed,
+the types of approvals, and which types of decision require which type of
+approval.
+
+###Voting
+
+Decisions regarding the project are made by votes on the primary project
+development mailing list (<a class="externalLink" href=
+"mailto:dev@apex.apache.org)">dev@apex.apache.org)</a>. Where necessary, PMC voting
```

Comment:
I would change "Where necessary" to "When involving personnel or other confidential matters" to make clear when it's necessary.